client/cypress/integration/query/parameter_spec.js

```diff
@@ -124,7 +135,8 @@ describe('Parameter', () => {
         SaveParameterSettings 
       `);
 
-      const now = new Date(2019, 0, 1).getTime(); // January 1, 2019 timestamp
+      const now = new Date();
+      now.setDate(1);
```

Comment:
Why wouldn't the original code work though?
Perhaps cause `cy.clock()` had better be called before `cy.visit()`? (https://docs.cypress.io/api/commands/clock.html#Behavior)

Reply:
I have no idea why, but `cy.clock` has a weird behavior, when writing the test if I put it before `cy.visit` the page wouldn't open :grimacing:.

I think it's a good option to check the format for now as it's not really necessary to have the date frozen.

Reply:
Yeah I couldn't find a way to fix the `cy.clock` behavior, so basically just made sure that a date change *within the current month and year* is applied correctly.

Reply:
As long as the test does what it should 👍